### PR TITLE
Restructure the cookbook with thematic and software sections

### PR DESCRIPTION
**Pull request by ceriottm**
Better structure allowing for overlap between topics and classification based on used tools.

### CODE REVIEW

**Comment by ceriottm** [comment]
Ready for review @SanggyuChong @DavideTisi

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
-that take more than 30 to 1 min seconds to execute. If you feel unsure if a
-contribution is suitable, feel free to contact one of the `support`_ person
-beforehand.
+that require more than a couple of minutes to execute.
+If you feel unsure if a
+contribution is suitable, feel free to open an issue on the
+`github repository <https://github.com/lab-cosmo/atomistic-cookbook>`_
+to discuss your ideas.
 
 Adding a new examples
```
Comment (SanggyuChong) [inline CONTRIBUTING.rst:22]: grammar fix needed

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -95,16 +100,22 @@ You can also build all examples (warning, this will take quite some time) with:
...
+.. marker-contrib-end
 
 Support
 -------
 
 If you still have problems adding your example to the repository, please feel
-free to contact one of the people
+free to contact one of the developers, e.g.
 
 `@agoscinski (Alexander Goscinski) <alexander.goscinski@epfl.ch>`_
```
Comment (SanggyuChong) [inline CONTRIBUTING.rst:135]: <3 <3 but I hope he is fine still being listed here

**Review by SanggyuChong — COMMENTED** [review]
Can't comment much on the reconfigurations of the cookbook, but regarding the thematic restructuring, I am happy with how it looks. Even better than I thought! I already think these two themes (topic/task and software) are enough, and nothing more should be added unless absolutely necessary.

Left a couple very minor comments along the way. I'll leave the actual approval to @DavideTisi to prevent blindly incorporating in some of these other changes I am not entirely up-to-date with. Cheers!

File: docs/src/software/chemiscope.sec [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+`python library <https://pypi.org/project/chemiscope/>`_ and
+used inside jupyter notebooks or sphinx documentations, which
+is how it's used in the cookbook examples. 
+You can check the `documentation <https://chemiscope.org/docs/>`_ and the 
+`github repository <https://github.com/lab-cosmo/chemiscope>`_.
+
+- examples/roy-gch/roy-gch
+- examples/gaas-map/gaas-map
+- examples/pi-metad/pi-metad
+- examples/path-integrals/path-integrals
```
Comment (DavideTisi) [inline docs/src/software/chemiscope.sec:20]: both pi-metad and path-integrals should not be here under chemiscope, becouse it seems they do not use it
```suggestion
```

**Review by DavideTisi — CHANGES_REQUESTED** [review]
also the linter is not happy

File: docs/src/software/chemiscope.sec (hunk 3 above)
Comment (ceriottm) [reply docs/src/software/chemiscope.sec:20]: yes they do use it
image

**Comment by DavideTisi** [comment]
Ok perfect

**Review by DavideTisi — APPROVED** [review]

**Pull request merged**